### ISSUE TITLE
ODOO_RC environment variable should be supported also

### ISSUE THREAD

**Issue by xtof-osd** [camptocamp/pytest-odoo#17]
In order to be compatible with odoo >= 10.0, ODOO_RC environment variable should be supported in addition to OPENERP_SERVER

**Comment by dcrocco** [comment]
What is this variable for?

**Comment by Jerther** [comment]
Sets the location of the configuration file:

```
$ echo $OPENERP_SERVER
/home/jerther/code/addons_test/openerp-server.conf
```

**Comment by dcrocco** [comment]
I mean ODOO_RC

**Comment by guewen** [comment]
It's released in 0.4.1 :) Thanks @xtof-osd
